src/main/java/util/LogUtil.java

```diff
@@ -0,0 +1,16 @@
+package util;
+
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+
+public class LogUtil {
+
+    /***
+     * Clear log file
+     */
+    public static void clearFile() throws IOException {
```

Comment:
This method is not reusable. If you need to clear a different file then you will have to create a separate method for that instead of reusing this, right?

Reply:
fixed